### ISSUE TITLE
Non-Modal BottomSheet 

### ISSUE THREAD

**Issue by Frizbi22** [lucacivale/Maui.BottomSheet#6]
I would like to request a feature to support non-modal BottomSheets in the library. Currently, the library provides excellent support for modal BottomSheets, but a non-modal option would be beneficial for use cases where the BottomSheet needs to remain persistent while allowing interaction with the underlying content.

**Comment by lucacivale** [comment]
Yes, this feature is definitely missing! Had a lot to do lately but I'll start soon working on this.
You are also welcome to create a PR if you want to :grin:

**Comment by softlion** [comment]
Ahhhhhhh there is no modal version ?
I'm using [The49](https://github.com/the49ltd/The49.Maui.BottomSheet) version of the bottomsheet library, but I prefer the architecture of your lib.

But I need the version that remains persistent like Frizbi22.

And also be able to specify the detents like in The49 would be great:
```
    <bottomSheet:BottomSheet.Detents>
        <bottomSheet:FullscreenDetent/>
        <bottomSheet:MediumDetent IsDefault="True" />
        <bottomSheet:HeightDetent Height="100" />
    </bottomSheet:BottomSheet.Detents>
```

**Comment by lucacivale** [comment]
No, currently there is no modal version. I know it's missing and it will be implemented in the future but I don't know when.
If you have any implementation ideas feel free to share them. This way we can speed up the implementation.

**Comment by edgiardina** [comment]
Upvoting

**Comment by lucacivale** [comment]
@softlion @edgiardina @Frizbi22 probably a bit late but I finally implemented this.
It's available in 9.0.6-pre.
